source/prefab/CP_Bug.h

```diff
@@ -0,0 +1,200 @@
+#ifndef EMP_CONFIG_PANEL_BUG_H
```

Comment:
File name? Did this get committed by accident?

Reply:
I added this file to help explain this issue (https://github.com/devosoft/Empirical/issues/307). I'll make a note about it in the file.

Reply:
Moved code to issue comment and deleted this file